### PR TITLE
update to beta tag 

### PR DESCRIPTION
**Pull request by rmatambo8**
upgrades the starter to have the new beta tag.

tested:

- builds correctly
- installs correctly
- no bugs in the console
- package.json file is updated and correct

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
   "dependencies": {
-    "gatsby": "^4.0.0-next.1",
-    "gatsby-plugin-image": "^2.0.0-next.1",
-    "gatsby-plugin-manifest": "^4.0.0-next.1",
-    "gatsby-plugin-offline": "^5.0.0-next.1",
-    "gatsby-plugin-react-helmet": "^5.0.0-next.1",
-    "gatsby-plugin-sharp": "^4.0.0-next.1",
-    "gatsby-source-filesystem": "^4.0.0-next.1",
-    "gatsby-transformer-sharp": "^4.0.0-next.1",
+    "gatsby": "^4.0.0-zz-next.1",
```
Comment (jxnblk) [inline package.json:8]: I would expect to see the `package-lock.json` updated as well. Did you use `npm` or `yarn`?

File: package.json [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
+    "gatsby-plugin-image": "^2.0.0-zz-next.1",
+    "gatsby-plugin-manifest": "^4.0.0-zz-next.1",
+    "gatsby-plugin-offline": "^5.0.0-zz-next.1",
+    "gatsby-plugin-react-helmet": "^5.0.0-zz-next.1",
+    "gatsby-plugin-sharp": "^4.0.0-zz-next.1",
+    "gatsby-source-filesystem": "^4.0.0-zz-next.1",
+    "gatsby-transformer-sharp": "^4.0.0-zz-next.1",
```
Comment (aghreed) [inline package.json:15]: Are all of these available at these dependency versions? 🤔 

Furthermore, I understood our task as just updating the `gatsby` package. Not the plugins / source / transformers.

File: package.json (hunk 1 above)
Comment (rmatambo8) [reply package.json:8]: yarn

File: package.json (hunk 2 above)
Comment (aghreed) [reply package.json:15]: Pulled this branch down and installed with no issues, so ignore my comment. 😄

File: package.json (hunk 1 above)
Comment (rmatambo8) [reply package.json:8]: i saw that it had a yarn.lock file so i used yarn and these were the only files that were added when i did `git add .`

Comment (jxnblk) [reply package.json:8]: Since this has both yarn and npm lock files. I'd recommend using the `npm i` CLI then `yarn import` (if needed)

**Comment by rmatambo8** [comment]
i think all issues mentioned above are resolved.

I also found out that gatsby serve does not work on brave browser.

**Review by jxnblk — APPROVED** [review]

**Review by aghreed — APPROVED** [review]

**Pull request merged**